uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Factory/RequestNotificationTemplate.php

```diff
@@ -2,9 +2,9 @@
 
 use CRM_HRLeaveAndAbsences_BAO_LeaveRequest as LeaveRequest;
 use CRM_HRLeaveAndAbsences_Service_LeaveRequestComment as LeaveRequestCommentService;
-use CRM_HRLeaveAndAbsences_Mail_SicknessRequestNotificationTemplate as SicknessRequestNotificationTemplate;
-use CRM_HRLeaveAndAbsences_Mail_TOILRequestNotificationTemplate as TOILRequestNotificationTemplate;
-use CRM_HRLeaveAndAbsences_Mail_LeaveRequestNotificationTemplate as LeaveRequestNotificationTemplate;
+use CRM_HRLeaveAndAbsences_Mail_Template_SicknessRequestNotificationTemplate as SicknessRequestNotificationTemplate;
```

Comment:
The idea of adding a new Template folder/namespace was to be able to remove it from the class name. So, instead of 
`CRM_HRLeaveAndAbsences_Mail_Template_SicknessRequestNotificationTemplate` you would have `CRM_HRLeaveAndAbsences_Mail_Template_SicknessRequestNotification`